backend/event_data/AUS/2991.csv

```diff
@@ -0,0 +1,85 @@
+event,date,gender,lifter,body_weight_(kg),snatch_lift_1,snatch_lift_2,snatch_lift_3,c&j_lift_1,c&j_lift_2,c&j_lift_3,best_snatch,best_c&j,total
+QWA State Masters Championships,2023-12-02,female,Lacey Bradford,44.10,13,15,-18,17,19,21,15,21,36
+QWA State Masters Championships,2023-12-02,female,Ava Nelson,47.10,14,15,-17,20,22,-24,15,22,37
+QWA State Masters Championships,2023-12-02,female,Mahalia Davidson,53.50,27,30,32,39,42,-45,32,42,74
+QWA State Masters Championships,2023-12-02,female,Julie Davis,55.00,24,-25,-25,-33,33,-35,24,33,57
+QWA State Masters Championships,2023-12-02,female,Natalie Marshall,57.50,58,61,65,75,80,82,65,82,147
+QWA State Masters Championships,2023-12-02,female,Lee-Ann Crooks,57.30,37,40,42,47,50,-53,42,50,92
+QWA State Masters Championships,2023-12-02,female,Diana Loy,58.20,28,-32,32,50,-55,55,32,55,87
+QWA State Masters Championships,2023-12-02,female,Erin Oberem,57.80,30,32,-34,38,40,-42,32,40,72
+QWA State Masters Championships,2023-12-02,female,Helena Hanlon,56.70,18,-19,20,27,28,30,20,30,50
+QWA State Masters Championships,2023-12-02,female,Rachel Mc Ewan,61.40,53,55,57,65,70,-75,57,70,127
+QWA State Masters Championships,2023-12-02,female,Jessica Malone,61.70,46,49,51,63,-67,67,51,67,118
+QWA State Masters Championships,2023-12-02,female,Holly Souter,60.40,49,52,-54,64,-67,-68,52,64,116
+QWA State Masters Championships,2023-12-02,female,Camielle Honrade,63.60,47,-50,-50,55,-58,-58,47,55,102
+QWA State Masters Championships,2023-12-02,female,Esther Moir,62.80,30,-32,-34,42,45,48,30,48,78
+QWA State Masters Championships,2023-12-02,female,Anna Thomson,66.80,65,68,71,85,89,-93,71,89,160
+QWA State Masters Championships,2023-12-02,female,Kaiya Ngahina Marsh,68.40,61,65,-68,75,80,82,65,82,147
+QWA State Masters Championships,2023-12-02,female,Erika Yamasaki,66.80,-63,63,67,78,,,67,78,145

```

Comment:
Should be 0s here. Not blank or the backend will shit the bed when it tries to parse it.